feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/SearchFilterRow.kt

```diff
@@ -126,11 +119,17 @@ private fun <T> FilterDropdown(
     items: List<T>,
     itemLabel: (T) -> String,
     onItemSelected: (T) -> Unit,
+    onMultiSelectFinished: (List<T>) -> Unit = {},
```

Comment:
Specifying default values for lambda parameters has some downsides: it can make it easy to forget to provide a value, and at a glance it’s not always clear whether a value is actually being passed. Therefore, unless there’s a specific reason, I’d prefer to remove default empty lambdas.